ngirimana/murikahomeltd
# #173268919 add logout user

> #### What does this PR do?
* Logout a user 
> #### Description of Task to be completed
- check timeout of user token
- delete data in local storage
- logout a user
> #### How should this be manually tested?
* clone repository
* run ``` cd murika-front-end``` in terminal
* checkout  **ft-logout-user-173268919**  branch by running ```git checkout ft-logout-user-173268919``` in terminal
* use 'npm install' to install all necessary dependencies
* Run 'npm start' to run the server 
* And click the login link
> #### Any background context you want to provide?
* you should install react  in your PC 
> #### What are the relevant pivotal tracker stories?
* [#173268919](https://www.pivotaltracker.com/story/show/173268919)
> #### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/38111965/84576612-60541e00-adb6-11ea-8d1c-a753c20e593c.png)
